src/Files.App/Views/Settings/AppearancePage.xaml

```diff
@@ -115,6 +115,18 @@
 					SelectedIndex="{x:Bind ViewModel.SelectedThemeIndex, Mode=TwoWay}" />
 			</local:SettingsBlockControl>
 
+			<!--  System Backdrop  -->
+			<local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsAppearanceSystemBackdrop}" HorizontalAlignment="Stretch">
```

Comment:
We don't need to specify "SettingsAppearance", especially since resources can be reused so we try to keep it generic.